etcd/etcd-live-http-to-https-migration.md

```diff
@@ -1,73 +1,73 @@
 
-If you've configured flannel, fleetd or other local components to use custom or 2379 port only, you have to configure them to use 4001 TCP port.
+If you've configured flannel, fleet, or other components to use custom ports, or 2379 only, they will be reconfigured to use port 4001.
 
-If your etcd configuration doesn't have configured 4001 client port, you have to enable it before we proceed with this manual. For example you use Cloud-Config to configure your CoreOS cluster. In this case you have to retrieve `ETCD_LISTEN_CLIENT_URLS` value from `/run/systemd/system/etcd2.service.d/20-cloudinit.conf` config file:
+If etcd isn't listening on port 4001, it must also be reconfigured. If you used cloud-config to spin up your machines, you can retrieve the `ETCD_LISTEN_CLIENT_URLS` value from `/run/systemd/system/etcd2.service.d/20-cloudinit.conf` to verify the etcd ports:
 
 ```sh
-grep ETCD_LISTEN_CLIENT_URLS /run/systemd/system/etcd2.service.d/20-cloudinit.conf
+$ grep ETCD_LISTEN_CLIENT_URLS /run/systemd/system/etcd2.service.d/20-cloudinit.conf
 Environment="ETCD_LISTEN_CLIENT_URLS=http://0.0.0.0:2379"
 ```
 
-Then create `/etc/systemd/system/etcd2.service.d/` directory if it doesn't exist and create [drop-in][drop-ins] file: `/etc/systemd/system/etcd2.service.d/25-insecure_localhost.conf`. Use value we've retrieved above and append `http://127.0.0.1:4001` URL at the end:
+In this case etcd is listening only on port 2379. We'll add port 4001 with a systemd [drop-in][drop-ins] unit file. Create the file `/etc/systemd/system/etcd2.service.d/25-insecure_localhost.conf`. In this file, write an excerpt that appends the new URL on port 4001 to the existing value we retrieved in the step above:
 
 ```
 [Service]
 Environment="ETCD_LISTEN_CLIENT_URLS=http://0.0.0.0:2379,http://127.0.0.1:4001"
 ```
 
-Then run `systemctl daemon-reload` and `systemctl restart etcd2` to restart etcd. Check cluster availability using [`etcdctl`][etcdctl] commands listed below:
+Run `systemctl daemon-reload` followed by `systemctl restart etcd2` to restart etcd. Check cluster status using the [`etcdctl`][etcdctl] commands:
 
 ```sh
-etcdctl member list
-etcdctl cluster-health
+$ etcdctl member list
+$ etcdctl cluster-health
 ```
 
-Repeat these steps on each CoreOS host which has etcd member.
+Repeat these steps on each etcd cluster member.
 
-### Generate TLS keypairs
+### Generate TLS Key Pairs
 
-Follow [generate self signed certificates][self-signed-ca] doc to generate self-signed certificates.
+Follow the guide to [generating self-signed certificates][self-signed-ca] to create the certificate/key pairs needed for each etcd cluster member.
```

Comment:
I understand self-signed is easy for users, but I'd like to either write in here, or link to, a recipe for setting up and running your own CA.  a) that's more secure b) it will track better with the process they follow to get a third-party-CA-signed cert.  That can be put in the linked doc maybe, at the top.  Then this link text would become just "generating certificates".